cc/src/test/scala/me/sgrouples/rogue/cc/RCcMetaExtSpec.scala

```diff
@@ -323,7 +325,8 @@ class MCcSpec extends FunSuite {
   }
   private val subtypedMeta = new SubtypedMeta
   assertEquals(
-    subtypedMeta.fieldNamesSorted,
+    // subtypedMeta.fieldNamesSorted,
+    Seq.empty,
```

Comment:
`fieldNamesSorted` was removed on the Scala 3 branch so I guess we don't need the test anymore either?